lib/config/migrations/custom/semantic-commits-migration.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { SemanticCommitsMigration } from './semantic-commits-migration';
+
+describe('config/migrations/custom/semantic-commits-migration', () => {
+  it('should migrate true to "enabled"', () => {
+    expect(SemanticCommitsMigration).toMigrate(
+      {
+        semanticCommits: true,
+      } as any,
```

Comment:
Just to confirm for my sake, the `any`s are explicitly required here due to the updated types in `RenovateConfig`?

Reply:
Yes, indeed, it is string now, but previously it was boolean apparently

Reply:
Yes, indeed, it is string now, but previously it was boolean apparently